hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -1136,6 +1141,49 @@ cdef class PureMarketMakingStrategy(StrategyBase):
                 order = orders[0]
                 if abs(order.price - price)/price >= self._hanging_orders_cancel_pct:
                     self.c_cancel_order(self._market_info, order.client_order_id)
+                # hanging orders older than max age are canceled and marked to be recreated.
+                elif order_age(order) > self._max_order_age:
+                    self.c_cancel_order(self._market_info, order.client_order_id)
+                    self._hanging_order_ids.remove(order.client_order_id)
+                    self._to_recreate_hanging_orders.append(order)
+
+    cdef c_did_cancel_order(self, object cancelled_event):
+        cdef:
+            list orders = [o for o in self._to_recreate_hanging_orders if o.client_order_id == cancelled_event.order_id]
+        if orders:
+            self.c_recreate_hanging_order(orders[0])
+
+    @property
+    def to_recreate_hanging_orders(self) -> List[LimitOrder]:
+        return self._to_recreate_hanging_orders
```

Comment:
Small matter, variable name suggest a return type of `bool`.
Might I suggest `_hanging_orders_to_recreate` instead

Reply:
agree, good suggestion.